### DISCUSSION TITLE
What is the word for the letter "Q" in the NATO phonetic alphabet?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#911] [Galaxy Trivia] [answered]
**Bonus:** Which 🕺🪩 are used as codewords in the alphabet?

**Comment by peterbaccaro** [comment, marked as answer]
The word for the letter "Q" in the NATO phonetic alphabet is "**Quebec**".

The dances used as a codewords in the NATO phonetic alphabet are **Foxtrot** and **Tango**.

**Comment by raeef1001** [comment]
The word for the letter "Q" in the NATO phonetic alphabet is "Quebec".

The dances used as a codewords in the NATO phonetic alphabet are Foxtrot and Tango.
